Makefile

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/make -f
+
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+COMMIT := $(shell git log -1 --format='%H')
+
+# don't override user values
+ifeq (,$(VERSION))
+  VERSION := $(shell git describe --tags)
+  # if VERSION is empty, then populate it with branch's name and raw commit hash
+  ifeq (,$(VERSION))
+    VERSION := $(BRANCH)-$(COMMIT)
+  endif
+endif
+
+PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
+DOCKER := $(shell which docker)
+BUILDDIR ?= $(CURDIR)/build
```

Comment:
We don't seem to ever use these variables. Delete?

Also, `PACKAGES_SIMTEST` seems specific to the cosmos-sdk. Does half-life need it?